templates/webapi/main/group_builds.html.ep

```diff
@@ -27,7 +27,7 @@
         </div>
         <div class="px-2 align-self-end flex-grow-1">
             % if ($max_jobs) {
-                %= include 'main/build_progressbar', max_jobs => $max_jobs, result => $build_res
+                %= include 'main/build_progressbar', max_jobs => $max_jobs, result => $build_res, class => ""
```

Comment:
I'd use single quotes so at least the quoting within that line is consistent.

Reply:
@ilausuch please attend this comment